### ISSUE TITLE
It was not possible to find any compatible framework version

### ISSUE THREAD

**Issue by solrevdev** [aws/aws-dotnet-deploy#224]
**Describe the bug**
The readme.md suggests that .NET Core 3.1 **or later** is required however I get the following error when running `dotnet aws --help`. 

I do have .NET 5.0.203 installed (macOS)

```bash
dotnet aws --help

It was not possible to find any compatible framework version
The framework 'Microsoft.AspNetCore.App', version '3.1.0' was not found.
  - The following frameworks were found:
      5.0.6 at [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]

You can resolve the problem by installing the specified framework and/or SDK.

The specified framework can be found at:
  - https://aka.ms/dotnet-core-applaunch?framework=Microsoft.AspNetCore.App&framework_version=3.1.0&arch=x64&rid=osx.10.15-x64
```

**Logs**
Logs obtained from CLI output.

```bash
 dotnet --list-sdks
5.0.203 [/usr/local/share/dotnet/sdk]

dotnet --info
.NET SDK (reflecting any global.json):
 Version:   5.0.203
 Commit:    383637d63f

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.15
 OS Platform: Darwin
 RID:         osx.10.15-x64
 Base Path:   /usr/local/share/dotnet/sdk/5.0.203/

Host (useful for support):
  Version: 5.0.6
  Commit:  478b2f8c0e

.NET SDKs installed:
  5.0.203 [/usr/local/share/dotnet/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 5.0.6 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 5.0.6 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download
```

```bash
 OS: 64bit Mac OS X 10.15.7 19H1030
 Kernel: x86_64 Darwin 19.6.0
 Uptime: 4h 59m
 Packages: 479
 Shell: zsh 5.8
 Resolution: 1920x1080
 DE: Aqua
 WM: Quartz Compositor
 WM Theme: Blue
 Disk: 11G / 480G (4%)
 CPU: Intel Core i7-3615QM @ 2.30GHz
 GPU: Intel HD Graphics 4000
 RAM: 8453MiB / 16384MiB
```

---

This is a :bug: bug report

**Comment by philasmar** [comment]
We are looking into this issue and will get back to you shortly.

**Comment by philasmar** [comment]
A fix has been released in in version 0.10.6.
